MIPX/MIPX.md

```diff
@@ -20,103 +20,111 @@ Replaces: N/A
-MIPXc2 defines a list of the digital assets owned by MakerDAO and should be managed by the MIP Editors. 
+MIPXc2 defines a list of the Intangible Assets owned by MakerDAO and should be managed by the MIP Editors. 
 
 #### vote.makerdao.com
 > **Asset Type:** Subdomain  
-> **Controller:** DUX Incubating Core Unit  
+> **Asset Owner:** DAI Foundation  
+> **Platform Manager:** DUX Incubating Core Unit (DUX email?)
+> **Content Manager:** GovAlpha Core Unit (GovAlpha@protonmail.com)
 > **Purpose:** Provide a safe and reliable way for Maker Governance to take part in the op-chain governance of MakerDAO.  
+> **Last Modification:** N/A (Grandfathered)  
 
 etc...
 
-### MIPXc3: Agreed Modification of Control
+### MIPXc3: Agreed Modification of Management
 
-An agreed modification of control refers to an uncontested and mutually agreed-upon transfer of control of a digital asset owned by MakerDAO. 
+In order for a Modification of Management to be valid under this component, it must meet the following requirements. In the event an asset has not yet been 'claimed' by any entity, the new managing entities must still follow the requirements listed in this component.
 
-A subproposal is not required for an agreed modification of control.
-
-In the event an asset has not yet been 'claimed' by any controller, the new controller must still follow the requirements listed in this component.
+**Transparency**  
+An agreed modification of control must be made publicly accessible to Maker Governance using the MakerDAO discourse forum. It must remain publicly accessible for at least one week before being considered valid and actioned by the relevant Asset Owner.
 
-In order for a modification of control to be valid under this component, it must meet the following requirements:
+The agreed Modification of Management must use the template located at **[MIPXc3-ForumPost-Template.md](MIPXc3-ForumPost-Template.md)**.
 
-**Transparency**  
-An agreed mofication of control must be made publicly accessible to Maker Governance using the MakerDAO discourse forum. It must remain publicly accessible for at least one week before being considered valid and actioned by the relevant 'Owner-Controller'.
+**Well Defined**  
+An agreed Modification of Management must include a listed purpose for the asset and reasoning for why the change in management is desirable. Additionally, the new managers must provide a public contact address (email) and commit to responding promptly to contact requests from lawful authorities and mandated actors.
 
-The agreed modification of control must use the template located at **[MIPXc3-ForumPost-Template.md](MIPXc3-ForumPost-Template.md)**.
+**Recorded**
+An agreed Modification of Management must include a link to an open pull request on the official MIPs GitHub repository which modifies MIPXc2 to take into account the agreed modification. The link to this pull request must be present in the discourse forum post as part of transparency requirements. 
```

Comment:
Previously it was stated that this was to be managed by MIP Editors.

Reply:
It is still managed by MIP editors, they have to merge the PR's.

Reply:
This is here to make their lives a little easier heh.